tools/karpenter-convert/README.md

```diff
@@ -9,7 +9,21 @@ It converts `v1alpha5/Provisioner` to `v1beta1/NodePool` and `v1alpha1/AWSNodeTe
 go install github.com/aws/karpenter/tools/karpenter-convert/cmd/karpenter-convert@latest
 ```
 
-## Usage:
+## Usage
+```console
+Usage:
+  karpenter-convert [flags]
+
+Flags:
+  -f, --filename strings               Filename, directory, or URL to files to need to get converted.
+  -h, --help                           help for karpenter-convert
+  -I, --ignore-defaults                Ignore defining default requirements when migrating Provisioners to NodePool.
```

Comment:
I'm not sure that I'm convinced here that we should have a one-letter short-value for this CLI option. I'm not convinced that this would be intuitive for users. Can we just keep the long-name?

Reply:
Dropped the short name